frontend/src/components/ConsultantRow.tsx

```diff
@@ -42,19 +42,19 @@ export default function ConsultantRows({
         <td
           className={`border-l-2 ${
             isListElementVisible
-              ? "border-l-secondary_default"
+              ? "border-l-secondary"
               : isRowHovered
-              ? "border-l-primary_default"
-              : "border-l-primary_l4"
+              ? "border-l-primary"
+              : "border-l-primary/5"
```

Comment:
Bare for at jeg skal lære: hva gjør `/5`  her på 48?

Reply:
Den setter opacity til fargen 😄 Så her vil du få primary fargen med 5% opacity, så du slipper å ha med border-l-primary og border-opacity-5: https://tailwindcss.com/docs/text-color#changing-the-opacity